source/clear-linux/guides/maintenance/enable-user-space.rst

```diff
@@ -56,23 +53,23 @@ utility:
 
 .. note::
 
-   The image we installed may not be the latest version of |CL| available on
+   The image installed may not be the latest version of |CL| available on
```

Comment:
OK to keep original per pronoun usage guidelines at https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/simple.